components/xesite-conv.tsx

```diff
@@ -7,15 +7,27 @@ export interface XesiteConvProps {
   children: ReactNode;
 }
 
+const widths = {
```

Comment:
xxx: some of my sticker images are non-square, this works around it to make things look correct.